demosplan/DemosPlanCoreBundle/Resources/config/GlobalConfig.php

```diff
@@ -44,6 +43,8 @@
 class GlobalConfig implements GlobalConfigInterface
 {
     private const PHASE_TRANSLATION_KEY_FIELD = 'translationKey';
+    private const DI_PLAN_COCKPIT_URL_KEY = 'DiPlanCockpit';
```

Comment:
I would not bind it to the project specific key but allow any url to have the placeholder. project specifics should not be used in core